doc/members/open_network.rst

```diff
@@ -69,7 +69,53 @@ The user can then make user RPCs, for example ``user_id`` to retrieve the unique
         "caller_id": 4
     }
 
-For each user CCF also stores arbitrary user-data in a JSON object, which can only be written to by members, subject to the standard proposal-vote governance mechanism. This lets members define initial metadata for certain users; for example to grant specific privileges, associate a human-readable name, or categorise the users. This user-data can then be read (but not written) by user-facing apps.
+User Data
+---------
+
+For each user, CCF also stores arbitrary user-data in a JSON object. This can only be written to by members, subject to the standard proposal-vote governance mechanism, via the ``set_user_data`` action. This lets members define initial metadata for certain users; for example to grant specific privileges, associate a human-readable name, or categorise the users. This user-data can then be read (but not written) by user-facing endpoints.
+
+For example, the C++ logging sample app has an endpoint (``/log/private/admin_only``) which uses user-data to restrict who is permitted to call it:
```

Comment:
Grammar savvy: "For example, the `/log/private/admin_only` endpoint in the C++ sample app uses..." would IMO read better.